test/specs/mainnet/01_sendToken/Arbitrum.spec.js

```diff
@@ -45,637 +50,556 @@ describe("The SDK, when sending a native token with arbitrum network on the Main
-        "The addition of transaction in the batch is not performed successfully."
-      );
-    }
-
-    try {
-      assert.isNotEmpty(
-        addTransactionToBatchOutput.requests[0].to,
-        "The To Address value is empty in the Batch Response."
-      );
-    } catch (e) {
-      console.log(e);
+    let output = await arbitrumMainNetSdk.getAccountBalances();
+    let native_balance;
+    let usdc_balance;
+    let usdt_balance;
+    let native_final;
+    let usdc_final;
+    let usdt_final;
+    let minimum_token_balance = 2;
+    let minimum_native_balance = 0.01;
+
+    for (let i = 0; i < output.items.length; i++) {
+      let tokenAddress = output.items[i].token;
+      if (tokenAddress === arbitrumNativeAddress) {
+        native_balance = output.items[i].balance;
+        native_final = utils.formatUnits(native_balance, 18);
+      } else if (tokenAddress === arbitrumUsdcAddress) {
+        usdc_balance = output.items[i].balance;
+        usdc_final = utils.formatUnits(usdc_balance, 6);
+      } else if (tokenAddress === arbitrumUsdtAddress) {
+        usdt_balance = output.items[i].balance;
+        usdt_final = utils.formatUnits(usdt_balance, 6);
+      }
     }
 
-    try {
-      assert.isNotEmpty(
-        addTransactionToBatchOutput.requests[0].data,
-        "The data value is empty in the Batch Reponse."
-      );
-    } catch (e) {
-      console.log(e);
+    if (
+      native_final > minimum_native_balance &&
+      usdc_final > minimum_token_balance &&
+      usdt_final > minimum_token_balance
+    ) {
+      runTest = true;
+    } else {
+      runTest = false;
     }
+  });
 
-    try {
-      assert.isNull(
-        addTransactionToBatchOutput.estimation,
-        "The estimation value is not null in the Batch Response."
-      );
-    } catch (e) {
-      console.log(e);
-    }
+  it("SMOKE: Perform the send native token on the arbitrum network", async () => {
+    if (runTest) {
+      let AddTransactionToBatchOutput;
 
-    // Estimating the batch
-    let estimationResponse;
-    let EstimatedGas_Estimate;
-    let FeeAmount_Estimate;
-    let EstimatedGasPrice_Estimate;
+      // Adding transaction to a batch
+      try {
+        AddTransactionToBatchOutput =
+          await arbitrumMainNetSdk.batchExecuteAccountTransaction({
+            to: toAddress,
+            value: value,
+          });
+      } catch (e) {
+        console.log(e);
+        assert.fail(
+          "The addition of transaction in the batch is not performed successfully."
+        );
+      }
 
-    try {
-      estimationResponse = await arbitrumMainNetSdk.estimateGatewayBatch();
-    } catch (e) {
-      console.log(e);
-      assert.fail("The estimation of the batch is not performed successfully.");
-    }
+      try {
+        assert.isNotEmpty(
+          AddTransactionToBatchOutput.requests[0].to,
+          "The To Address value is empty in the Batch Response."
+        );
+      } catch (e) {
+        console.log(e);
```

Comment:
Remember to use `console.error` when you are reporting an error

Reply:
Please change this throughout the code

Reply:
updated the code in all the test cases as per the comment.